docs/install/_linux.rst

```diff
@@ -23,6 +23,32 @@ Use the following commands in `Terminal <https://help.ubuntu.com/community/Using
     sudo apt-get update
     sudo apt-get install kolibri
 
+kolibri-server (pre-release)
+****************************
+
+We are finalizing the release of an optimized version of Kolibri for Linux systems called **kolibri-server**. This version is recommended for:
+
+* Clear server-client setup for implementations with many client devices
+* When the server device has more than one CPU core
+
+**kolibri-server** has some specific features like **static content cache** and **optimized usage of the CPU processing power** in multi core devices. Typical use case would be a *Raspberry Pi 3+* device with 4 CPU cores, which will perform much better and faster with **kolibri-server** compared to base Kolibri.
+
+To install **kolibri-server** follow these steps.
+
+#. Run these commands in the Terminal:
+
+	.. code-block:: bash
+
+		sudo apt-get install software-properties-common python-software-properties
+		sudo add-apt-repository ppa:learningequality/kolibri-proposed
+		sudo apt-get update
+		sudo apt-get install kolibri-server
```

Comment:
I think we should add a warning that this also switches the setup to use the latest built Kolibri pre-release?

Reply:
Righto, will add! :+1:

Reply:
Have added a suggestion for a re-location of this and a different heading further up. Basically, I don't think that `kolibri-server (pre-release)` is a very telling name :)